feature_campaigns/src/main/java/cm/aptoide/pt/feature_campaigns/AptoideMMPCampaign.kt

```diff
@@ -0,0 +1,30 @@
+package cm.aptoide.pt.feature_campaigns
+
+class AptoideMMPCampaign(
```

Comment:
I don't see it's used anywhere it this PR.
Let's add it in the PR for Aptoide MMP, not in this one.